mne_icalabel/iclabel/label_components.py

```diff
@@ -57,7 +57,7 @@ def iclabel_label_components(inst: Union[BaseRaw, BaseEpochs], ica: ICA, inplace
 
         # add labels to the ICA instance
         for idx, (_, mne_label) in enumerate(ICLABEL_LABELS_TO_MNE.items()):
-            auto_labels = np.argwhere(argmax_labels == idx)
+            auto_labels = list(np.argwhere(argmax_labels == idx).flatten())
```

Comment:
@adam2392 Note that I changed the format from an array to a list here. Let me know if that causes an issue in your opinion.